### PR TITLE
Add Pipeline example configs

### PR DESCRIPTION
**Pull request by hisaac**

### CODE REVIEW

**Review by daniCsorba — COMMENTED** [review]
looks good to me! :)

**Comment by godrei** [comment]
I'm not sure if the workflow-recipes repo is the best for storing the Pipeline examples bitrise.ymls.
A recipe is more like the full guide we created for the ios and android use cases and not just a single bitrise.yml.
I feel like, we should either migrate the guide to a recipe or store the bitrise.yml somewhere else.

**Comment by hisaac** [comment]
I've updated the yaml files to be in recipe format. You can see rendered versions of the recipes here:

- [android-parallel-testing-unit-test-shards.md](https://github.com/bitrise-io/workflow-recipes/blob/add-pipeline-example-configs/recipes/android-parallel-testing-unit-test-shards.md)
- [android-parallel-ui-tests-on-multiple-devices.md](https://github.com/bitrise-io/workflow-recipes/blob/add-pipeline-example-configs/recipes/android-parallel-ui-tests-on-multiple-devices.md)
- [android-parallel-unit-and-ui-tests.md](https://github.com/bitrise-io/workflow-recipes/blob/add-pipeline-example-configs/recipes/android-parallel-unit-and-ui-tests.md)

File: recipes/android-parallel-testing-unit-test-shards.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+```yaml
+format_version: "11"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: android
+
+meta:
+  bitrise.io:
+    stack: linux-docker-android-20.04
+    machine_type_id: elite
```
Comment (godrei) [inline recipes/android-parallel-testing-unit-test-shards.md]: All the bitrise.ymls: Elite machines can be a problem, for customers in hobby plans without Elite machines.

**Comment by godrei** [comment]
Do not forget to add the new recipes to the main RADME.md, probably all Pipeline recipes should go under a new section (Pipeline or Pipeline example or somthing similar)

**Comment by hisaac** [comment]
Good call, I didn't notice there was a list there. Will update.

File: recipes/android-parallel-testing-unit-test-shards.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+1. Select the `main` branch to scan.
+1. Wait for the project scanner to complete.
+1. Enter `app` as the specified module.
+1. Enter `debug` as the specified variant.
+1. Continue through the prompts as normal — no changes are needed.
+1. Open the new Bitrise project’s Workflow Editor.
+1. Go to the **bitrise.yml** tab, and replace the existing yaml contents with the contents of the example `bitrise.yml` below.
+1. Click the **Start/Schedule a Build** button, and select the `pipeline_unit_test` option in the **Workflow, Pipeline** dropdown menu at the bottom of the popup.
+
+```yaml
```
Comment (godrei) [inline recipes/android-parallel-testing-unit-test-shards.md:32]: Based on the [recipe template](https://github.com/bitrise-io/workflow-recipes/blob/add-pipeline-example-configs/contribution/recipe-template.md) the bitrise.yml should go under the `bitrise.yml` section.

File: recipes/android-parallel-testing-unit-test-shards.md (hunk 1 above)
Comment (hisaac) [reply recipes/android-parallel-testing-unit-test-shards.md]: Whoops, I updated that in the Confluence doc but forgot to update that here.

File: recipes/android-parallel-testing-unit-test-shards.md (hunk 2 above)
Comment (hisaac) [reply recipes/android-parallel-testing-unit-test-shards.md:32]: Thanks, missed that one

File: recipes/android-parallel-testing-unit-test-shards.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+# Unit test sharding by module
```
Comment (godrei) [inline recipes/android-parallel-testing-unit-test-shards.md]: Based on the available recipes the platform should be present in the recipe title (`(Android)`)

**Pull request merged**